demo/seed-devnet.sh

```diff
@@ -97,12 +88,14 @@ function publishReferenceScripts() {
     --cardano-signing-key devnet/credentials/faucet.sk
 }
 
-seedFaucet "alice" 1000000000 # 1000 Ada to commit
-seedFaucet "alice" 100000000 "fuel" # 100 Ada marked as "fuel"
-seedFaucet "bob" 500000000 # 500 Ada to commit
-seedFaucet "bob" 100000000 "fuel" # 100 Ada marked as "fuel"
-seedFaucet "carol" 250000000 # 250 Ada to commit
-seedFaucet "carol" 100000000 "fuel" # 100 Ada marked as "fuel"
+echo >&2 "Fueling up hydra nodes of alice, bob and carol..."
+seedFaucet "alice" 20000000 # 20 Ada fuel available to the node
+seedFaucet "bob" 20000000 # 20 Ada fuel available to the node
+seedFaucet "carol" 20000000 # 20 Ada fuel available to the node
+echo >&2 "Distributing funds to alice, bob and carol..."
+seedFaucet "alice-funds" 100000000 # 100 Ada to commit
+seedFaucet "bob-funds" 50000000 # 50 Ada to commit
+seedFaucet "carol-funds" 25000000 # 25 Ada to commit
```

Comment:
@v0d1ch Updated this as you seem to have confused the comments. Also, I made every node have constant 20 ADA fuel (which should be enough for a single head life cycle)